google/cloud/storage/bucket.py

```diff
@@ -266,6 +286,18 @@ def number_of_newer_versions(self):
         """Conditon's 'number_of_newer_versions' value."""
         return self.get("numNewerVersions")
 
+    @property
+    def days_since_custom_time(self):
+        """Conditon's 'days_since_custom_time' value."""
+        return self.get("daysSinceCustomTime")
+
+    @property
+    def custom_time_before(self):
+        """Conditon's 'custom_time_before' value."""
+        before = self.get("customTimeBefore")
+        if before is not None:
+            return datetime_helpers.from_iso8601_date(before)
```

Comment:
Not sure about the setter method for all this properties, because we don't have a method for other properties as well which  defined before. Please let me know if needed, so need to create setter method for all the properties of `LifecycleRuleConditions` class.

Reply:
Thanks both, looks like setters weren't defined at this level. It will be filed as a feature request for now. It doesn't block customers from using the feature.